scancode_api/pom.xml

```diff
@@ -40,6 +40,12 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+
+		<dependency>
```

Comment:
Yup, this is why you don't need to have MongoDB installed. Maven is grabbing it for you.